components/usage/.gitignore

```diff
@@ -0,0 +1 @@
+apikeys
```

Comment:
Interesting -- is placing an untracked, secret file next to the `cmd` the recommended way to provide credentials to a Go component? 👀 (I haven't worked on Go component secrets before.)

Reply:
I think this way works well here. When the component runs in a pod it takes the credentials file in the same way (file on disk, location passed via `--api-keys-file`).

How would you expect the necessary api keys to be provided when running the usage component locally?

Reply:
Makes sense. I think @easyCZ is keen to move away from requiring preview environments to develop components. When working on the usage component I think the intention is to run it locally, port forwarding to a database as necesary.